addons/web/static/src/core/datetime/datetime_picker.scss

```diff
@@ -26,6 +26,10 @@
         &.o_active_start,
         &.o_active_end {
             color: var(--DateTimePicker__Today-highlight-color);
+            
+            &, &:hover{
+                background-color: map-get($theme-colors, primary);
```

Comment:
The component makes use of CSS variables.
I'd prefer to keep using the same approach.

Reply:
If I understand correctly, we should use the action color for the date picker? But we don't have css variable for that color. Can't we use `$o-action` then? + `--bg-opacity` doesn't affect colors somehow

Reply:
>  Can't we use $o-action then

Yes, in this case we can 👍 

> --bg-opacity doesn't affect the colors somehow

`--bg-opacity` can be used for `bg-[x]` classes only.